src/SkyWalking.AspNet/SkyWalkingOptions.cs

```diff
@@ -55,5 +55,23 @@ public class SkyWalkingOptions
         /// Zero or minus value means no limit.
         /// </summary>
         public int PendingSegmentsLimit { get; set; } = 300000;
+        
+        /// <summary>
+        /// Set your own token to active auth
+        /// </summary>
+        public string Authentication
```

Comment:
Is it better rename `Authentication` to `AuthenticationToken` ?

Reply:
Refer to the [Set Token](https://github.com/apache/incubator-skywalking/blob/master/docs/en/Token-auth.md#set-token). I prefer to use the same config name.